src/stores/video.ts

```diff
@@ -269,9 +353,44 @@ export const useVideoStore = defineStore('video', () => {
     await tempVideoChunksDB.clear()
   }
 
+  // Used to download a file from the video recovery database
+  const downloadTempVideo = async (hashes: string[]): Promise<void> => {
+    console.debug(`Downloading ${hashes.length} video chunks from the temporary database.`)
+
+    for (const hash of hashes) {
+      const fileNames = await tempVideoChunksDB.keys()
+      const maybeFiles = await Promise.all(
+        fileNames.map(async (filename) => ({
+          blob: await tempVideoChunksDB.getItem(filename),
+          filename,
+        }))
+      )
+      const files = maybeFiles.filter((file) => file.blob !== undefined && file.filename.includes(hash))
+
+      if (files.isEmpty()) {
+        Swal.fire({ text: 'No files found.', icon: 'error' })
+        return
+      }
+
+      if (files.length === 1) {
+        saveAs(files[0].blob as Blob, files[0].filename)
+        return
+      }
+
+      const zipWriter = new ZipWriter(new BlobWriter('application/zip'))
+
+      for (const { filename, blob } of files) {
+        await zipWriter.add(filename, new BlobReader(blob as Blob))
+      }
+
+      const blob = await zipWriter.close()
+      saveAs(blob, `Cockpit-Temp-Video-Chunks-${hash}.zip`)
+    }
+  }
+
```

Comment:
Isn't most of the code here duplicated with the `downloadFilesFromVideoDB` method? Could we reuse it?

Reply:
The second function was used inside the old Video Configurations Page and was intended to allow the user to download all temporary chunks from the DB. 
Since the user can now download the chunks from selected videos and can also select all videos and download its chunks, this function isn't necessary anymore.
I'll remove it from the video store and remove its calling from the video configurations page.